www/source/community.html.slim

```diff
@@ -7,50 +7,95 @@ section.community--hero
   .row
     .columns.medium-12
       .community--hero--content
-        h1.community--hero--heading Let's build together
+        h1.community--hero--heading Habitat Open Source Community
         p.community--hero--text
-          | Habitat is an open source project created and supported
-            by a passionate group of people. Our vision is grand — to bring automation
-            to places previously unconsidered.
+          | We are a passionate group of people, dedicated to bringing automation to places previously unconsidered.
+            Let’s connect at upcoming meetups, training and conferences!
         .row
           .columns.medium-12
             ul.no-bullet
               li
-                a.button.cta href="http://slack.habitat.sh" target="_blank" Join us on Slack
+                a.button.cta href="http://slack.habitat.sh" target="_blank" Join Habitat Slack
               li
-                a.button.cta href="https://forums.habitat.sh/" target="_blank" Visit the Forums
-              li
-                a.button.cta href="https://events.chef.io/events/categories/habitat/" target="_blank" Meet at Events
-              li
-                a.button.cta href="https://blog.chef.io/category/habitat/" target="_blank" Read Blog Posts
+                a.button.outline href="/docs/contribute-help-build/" Contribute to Habitat
 
-    .community--hero--graphic
-      img[src="/images/homepage/container-icon-3.svg"
-        onerror="this.src='/images/homepage/container-icon-3.png'" alt=""]
+section.community--events
+  .row
+    .columns.medium-12
+      .community--events--content
+        h2 Upcoming Events
+        .row
+          .community--events--card.columns.medium-6.medium-push-3
+            .community--events--card-heading.heading
+              ul.no-bullet
+                li Meetup
+            .body-content
+              h4 ChefConf 2017
+              p
+                | Join us at ChefConf to learn aobut all things Habitat. There will be several presentations
+                  during the conference in addition to ongoing discussions in the Habitat zone.
+              .article-detail--meta
+                ul.no-bullet
+                  li
+                    img[src="/images/icons/icon-calendar.svg"
+                      onerror="this.src='/images/icons/icon-calendar.png'" alt="calendar icon"]
```

Comment:
This is safe, but I don't think it's necessary anymore; support for SVGs as `img` references is [pretty solid](http://caniuse.com/#feat=svg-img).